management-api-agent-dse-6.9/README.md

```diff
@@ -0,0 +1,80 @@
+# Management API with DSE
+
```

Comment:
This is just a copy-paste from the dse-6.8 agent project. The content should be the same, but with the version referenced changed to 6.9.